parlai/scripts/convert_data_to_parlai_format.py

```diff
@@ -19,19 +19,32 @@
 from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
 from parlai.core.worlds import create_task
 from parlai.core.utils import msg_to_str
-
 import random
+import tempfile
+from tqdm import tqdm
 
 
 def dump_data(opt):
     # create repeat label agent and assign it to the specified task
     agent = RepeatLabelAgent(opt)
     world = create_task(opt, agent)
     ignorefields = opt.get('ignore_fields', '')
+    if opt['outfile'] is None:
+        outfile = tempfile.mkstemp(
+            prefix='convert_data_to_parlai_output',
+            suffix='.txt')[1]
```

Comment:
what was wrong with /tmp/dump?  what does this do?

Reply:
By default it was always using the same file, so overwriting the previous thing. Now file with prefix convert_data_to_parlai_output and some random suffix is created in /tmp folder by default if opt['outfile'] is not specified by the user. The full file name will be printed during the execution

Reply:
random is kinda weird but ok..  it could have the name of the actual task from "-t" and the type from "dt" in the name if u like?

Reply:
agreed, will make /tmp/task_datatype_random.txt, the random is only suffix part such that if you run couple of times with some changed code inside of another agent or so and you forgot to specify - your old results will not be overwritten :)